### DISCUSSION TITLE
SoundSwitch v6.11.0

### DISCUSSION THREAD

**Discussion by Belphemur** [Belphemur/SoundSwitch#1513] [Releases]
Major changes are:
- You can now configure if you want only one or multiple notification visible at the same time (old vs new behavior)
- How long the notification stays on the screen
- A rework of the systray menu to streamline and improve user experience
- A new Troubleshoot tab in the settings containing tools to provide logs and other help resources
- Swedish language added

### Enhancements

* **banner:** Added Center Position ([60e5954](https://github.com/Belphemur/SoundSwitch/commit/60e59543244e3ae0021358270ec70322273c7f28))
* **center:** Middle of the screen/center position for accessibility ([9053231](https://github.com/Belphemur/SoundSwitch/commit/9053231f4c3dfc3f21b80f01767a4090da2a4c5b)), closes [#1466](https://github.com/Belphemur/SoundSwitch/issues/1466)
* **language:** Add Swedish translations ([27ef51f](https://github.com/Belphemur/SoundSwitch/commit/27ef51fe7911f047d2a79e9171a35bdf035fe593)), closes [#1431](https://github.com/Belphemur/SoundSwitch/issues/1431)
* **profile:** improve description of profile and let the text auto wrap ([551c41b](https://github.com/Belphemur/SoundSwitch/commit/551c41b97c7cdd08a8ed2e4e94efbd7cc5791d5c)), closes [#1433](https://github.com/Belphemur/SoundSwitch/issues/1433)
* **settings): Troubleshooting; refactor(trayIcon:** Updated Tray Menu ([f132a89](https://github.com/Belphemur/SoundSwitch/commit/f132a894483f06056cfdf85690aabfdc597b2ca3))
* **updater:** auto retry when can't download the file ([51bac9c](https://github.com/Belphemur/SoundSwitch/commit/51bac9c4d027d163e7516e3537eff812bcde1741))

### Languages

* **Chinese (Simplified):** Translated Settings using Weblate ([c08e449](https://github.com/Belphemur/SoundSwitch/commit/c08e4499a498c8cd31a79fd232c1168f7b5d1d55))
* **Croatian:** Translated About using Weblate ([b16e510](https://github.com/Belphemur/SoundSwitch/commit/b16e510afd7003698505ced4c7b30e5a0229fa83))
* **Croatian:** Translated Settings using Weblate ([4384e8c](https://github.com/Belphemur/SoundSwitch/commit/4384e8cbe03bac434bfcd5b97e78c19d08100b50))
* **Danish:** Translated Settings using Weblate ([6d5d619](https://github.com/Belphemur/SoundSwitch/commit/6d5d619e44af73bf63d8859524e279d4e1651be5))
* **Dutch:** Translated Settings using Weblate ([7c48d01](https://github.com/Belphemur/SoundSwitch/commit/7c48d01e0e15d9a4d9b345be304bb797231b8524))
* **Dutch:** Translated Settings using Weblate ([3103328](https://github.com/Belphemur/SoundSwitch/commit/31033283a4b13a630f2c5311945b40937ccb22f3))
* **Dutch:** Translated Update Download using Weblate ([681c508](https://github.com/Belphemur/SoundSwitch/commit/681c5083e763a0f09555c4b5daf72a21b086f91a))
* **English:** Translated Settings using Weblate ([c661601](https://github.com/Belphemur/SoundSwitch/commit/c66160145d273125c96b1cb0424198bdbe7a8315))
* **English:** Translated Settings using Weblate ([f30acf6](https://github.com/Belphemur/SoundSwitch/commit/f30acf6a133ff674149a0991e0fc7d8607e830f8))
* **French:** Translated Settings using Weblate ([765c837](https://github.com/Belphemur/SoundSwitch/commit/765c837d84b6309530f6541984480b8f9e26f0ae))
* **Hebrew:** Translated Settings using Weblate ([8ada5b1](https://github.com/Belphemur/SoundSwitch/commit/8ada5b1f6bf62bd43eac9fb1a20cee805f313d93))
* **Hebrew:** Translated Settings using Weblate ([36ba8bc](https://github.com/Belphemur/SoundSwitch/commit/36ba8bc02c7318f62fddffaa1ee098a0bc96fffe))
* **Hebrew:** Translated Settings using Weblate ([8e2ad75](https://github.com/Belphemur/SoundSwitch/commit/8e2ad75251600e9f4b5adb83d15ee70fabefa421))
* **Italian:** Translated Settings using Weblate ([5d306f2](https://github.com/Belphemur/SoundSwitch/commit/5d306f2711ce667322de2571fa8a7a7f52db9108))
* **Italian:** Translated Settings using Weblate ([6e9d2b5](https://github.com/Belphemur/SoundSwitch/commit/6e9d2b5fdc9421ba0483fbc8a296853fbb7280b8))
* **Japanese:** Translated About using Weblate ([5cedb13](https://github.com/Belphemur/SoundSwitch/commit/5cedb13c687a976c676efb6f565d66fe7e1737a3))
* **Japanese:** Translated About using Weblate ([8abbb54](https://github.com/Belphemur/SoundSwitch/commit/8abbb54ff10dbb069feee9dd79930b775c7e81e2))
* **Japanese:** Translated Settings using Weblate ([4ed10a0](https://github.com/Belphemur/SoundSwitch/commit/4ed10a0bf7d75582433b0360d9a7e100e3b94c78))
* **Japanese:** Translated Settings using Weblate ([2b89880](https://github.com/Belphemur/SoundSwitch/commit/2b89880ea87a7696b2fd26c5fb1ef9267afae73f))
* **Japanese:** Translated Settings using Weblate ([94629ce](https://github.com/Belphemur/SoundSwitch/commit/94629ceb1334521dd7873c6e2e31c789b33ef7aa))
* **Japanese:** Translated Settings using Weblate ([27e14d9](https://github.com/Belphemur/SoundSwitch/commit/27e14d90a74da9403668d82da9c6ee631aaba9f7))
* **Japanese:** Translated Settings using Weblate ([d82cb2b](https://github.com/Belphemur/SoundSwitch/commit/d82cb2be155735f8d5a0244b0feb9372bd3cb606))
* **Japanese:** Translated Settings using Weblate ([8aa4f82](https://github.com/Belphemur/SoundSwitch/commit/8aa4f82d41ae62f673099b99557cf410aeb7bc29))
* **Japanese:** Translated Settings using Weblate ([00ed06c](https://github.com/Belphemur/SoundSwitch/commit/00ed06c169c796a2b3eb2c63ede84fc7d347c85a))
* **Japanese:** Translated Settings using Weblate ([fde374e](https://github.com/Belphemur/SoundSwitch/commit/fde374e2c689a5c9e3ec83dac8fa7d710f3ca945))
* **Japanese:** Translated Tray Icon using Weblate ([8c97167](https://github.com/Belphemur/SoundSwitch/commit/8c97167020d4cd698ef15d7af555a0632c8945d0))
* **Japanese:** Translated Tray Icon using Weblate ([5f0de1b](https://github.com/Belphemur/SoundSwitch/commit/5f0de1bc1a06d5ee41ae348cf88a6b80eb992853))
* **Japanese:** Translated Update Download using Weblate ([872a51f](https://github.com/Belphemur/SoundSwitch/commit/872a51f290faa7d3b847dfcb83b40315f1e70a4c))
* **Japanese:** Translated Update Download using Weblate ([525ae92](https://github.com/Belphemur/SoundSwitch/commit/525ae92dfb2d8d996346ab5e56de115090604006))
* **Spanish:** Translated Settings using Weblate ([dd90865](https://github.com/Belphemur/SoundSwitch/commit/dd9086585df1a89233adc0f58de5eb4f3fbbfc4a))
* **Spanish:** Translated Settings using Weblate ([dc8fe47](https://github.com/Belphemur/SoundSwitch/commit/dc8fe472ac9dce2ab3f574419e67d8f50cd71a59))
* **Spanish:** Translated Settings using Weblate ([1c343e6](https://github.com/Belphemur/SoundSwitch/commit/1c343e65f2257f701208e32f26ea8400111afb8e))
* **Spanish:** Translated Settings using Weblate ([57ff02b](https://github.com/Belphemur/SoundSwitch/commit/57ff02bac55fc7238d7c2fd5786cc191b1c3b1d3))
* **Swedish:** Translated Settings using Weblate ([33dbc50](https://github.com/Belphemur/SoundSwitch/commit/33dbc502ad3ed5b3bd55acfa2f9f807cc5402566))

### Bug Fixes

* **banner/quickmenu:** Don't show notification or quick menu in the ALT+TAB menu ([5312f64](https://github.com/Belphemur/SoundSwitch/commit/5312f64a79f9657ad0146c2a0346833bdd2888bc)), closes [#1475](https://github.com/Belphemur/SoundSwitch/issues/1475)
* **banner:** banner crashing randomnly ([8ffa6d9](https://github.com/Belphemur/SoundSwitch/commit/8ffa6d9dc5771551a5ab16d3c4d8f1f587324aee))
* **ci:** downgrade conventianl commits ([c54c3e0](https://github.com/Belphemur/SoundSwitch/commit/c54c3e0c3732fc4096e28a627a65e8f5d78dc424))
* **ci:** fix dev version calculation ([81253d4](https://github.com/Belphemur/SoundSwitch/commit/81253d4652e85b597bdea14caf392c6d13085b8f))
* dev version calculation ([8fecb98](https://github.com/Belphemur/SoundSwitch/commit/8fecb98aca7ba7a62bad9cbad138cc4c5f295949))
* **localization:** Updated Settings and TrayIcon Entries ([145f98d](https://github.com/Belphemur/SoundSwitch/commit/145f98d4542f0ebaaa285b153cc507d7fb61859c))
* **profile:** fix crashing when switching profile ([0bf93e6](https://github.com/Belphemur/SoundSwitch/commit/0bf93e6879b03e6a5369646e87a86b652f96a60c)), closes [#1481](https://github.com/Belphemur/SoundSwitch/issues/1481)
* **profiles:** Readjusted items in Profiles tab ([75c4ea0](https://github.com/Belphemur/SoundSwitch/commit/75c4ea092a074e15238eed23a82be53bae4664cd))
* **profile:** used icon for profile ([8a8237b](https://github.com/Belphemur/SoundSwitch/commit/8a8237b4c53bd375304edeaac41cebf45e09a83a))
* **quickmenu:** Fix crash with QuickMenu having disposed icon ([6a79ec9](https://github.com/Belphemur/SoundSwitch/commit/6a79ec9f03c4f3397c8bb32bbb655bc85865cec0))
* **quickmenu:** possible crash when quick menu triggered while disappearing ([bd51528](https://github.com/Belphemur/SoundSwitch/commit/bd51528ea4e23f22c5f96b59183d2afe323e15c3))
* **Settings:** Fix issue where the settings close button could overlap with text. ([f5ceed7](https://github.com/Belphemur/SoundSwitch/commit/f5ceed7cceecc5ec0fc29222496e3c56a0c41ed3)), closes [#1448](https://github.com/Belphemur/SoundSwitch/issues/1448)

### Features

* **banner:on-screen-time:** Settings to change how long the banner stays on the screen ([41644f5](https://github.com/Belphemur/SoundSwitch/commit/41644f520ebd63fff292e9661db597fdba806c82)), closes [#1467](https://github.com/Belphemur/SoundSwitch/issues/1467)
* **notification:** Make the number of banner notification on the screen configurable ([cc15647](https://github.com/Belphemur/SoundSwitch/commit/cc15647b42f39265edc0f6186b0bb64ee93578d6))

<hr /><em>This discussion was created from the release <a href='https://github.com/Belphemur/SoundSwitch/releases/tag/v6.11.0'>SoundSwitch v6.11.0</a>.</em>

**Comment by MasterBlastered** [comment]
I love Game Backup Monitor!!
2 Suggestions,
Have an option for a minimal size UI I can leave up on my desktop that takes up less space. Maybe have it user selectable to a single specific game, Include Backup, Restore, Exit minimal mode, and maybe Edit that would also restore GBM to normal size out of this minimal view. Maybe also include File, Setup, Tools, Help which would also switch to normal mode.
Allow a dark mode. Many programs come with this these days. Yours is my only non dark mode app.

**Comment by vertigo220** [comment]
I have to say I didn't like the multiple banner change in this latest version. Spent at least a few minutes trying to figure out what was going on, thinking maybe something changed with Windows updates (just ran a bunch at the same time I updated this, so thought maybe it was a Windows notification) or with EarTrumpet (tested closing it), and took a while to find the new setting responsible. Probably should be defaulted to the old behavior of a single notification, especially since the second banner only shows the same thing as the main banner (though half of it since it just shows the playback device) making it seem like it's something else doing it, since it wouldn't make sense for this, or any, program to show the same thing twice. Perhaps putting the SoundSwitch icon in the banner(s) would help as well, so it's more clear they're (both) from this app. Though at this point it may be moot, as most users have likely switched over and figured this out.
